spec/model/associations_spec.rb

```diff
@@ -2,6 +2,71 @@
 require File.join(File.dirname(__FILE__), "../spec_helper.rb")
 
 describe Her::Model::Associations do
+
+  context "building and creating association data" do
```

Comment:
Let's keep the `setting associations` specs up top. It's useful to first understand how association definitions are stored on the model before diving into how they influence build & creation.